installer/install.iss

```diff
@@ -846,12 +846,12 @@ begin
                         end;
                     'credential.helper':
                         case Value of
-                            'manager': RecordInferredDefault('Use Credential Manager','Core');
-                            'manager-core': RecordInferredDefault('Use Credential Manager','Core');
+                            'manager': RecordInferredDefault('Use Credential Manager','Enabled');
+                            'manager-core': RecordInferredDefault('Use Credential Manager','Enabled');
                         else
                             begin
-                                if EndsWith(Value, 'manager-core') then
-                                    RecordInferredDefault('Use Credential Manager','Core')
+                                if EndsWith(Value,'manager-core') or EndsWith(Value,'manager') then
```

Comment:
So this means both manager and manager-core are resolved to the new GCM? Cool!

Reply:
Yes!